Update github-enterprise-managed-user-tutorial.md (#75290)
Azure/azure-content: articles/active-directory/saas-apps/github-enterprise-managed-user-tutorial.md

## Configure and test Azure AD SSO for GitHub Enterprise Managed User

To configure and test Azure AD SSO with GitHub Enterprise Managed User, perform the following steps:

> 1. **[Configure Azure AD SSO](#configure-azure-ad-sso)** - to enable your users to use this feature.

Reviewer: ```suggestion
1. **[Configure Azure AD SSO](#configure-azure-ad-sso)** - to enable SAML Single Sign On in your AAD tenant.
```

> 1. **[Configure GitHub Enterprise Managed User SSO](#configure-github-enterprise-managed-user-sso)** - to configure the single sign-on settings on application side.

Reviewer: ```suggestion
1. **[Configure GitHub Enterprise Managed User SSO](#configure-github-enterprise-managed-user-sso)** - to configure the single sign-on settings in your GitHub Enterprise.
```

    `https://github.com/enterprises/<ENTITY>/sso`

1. On the **Set up single sign-on with SAML** page, in the **SAML Signing Certificate** section,  find **Certificate (Base64)** and select **Download** to download the certificate and save it on your computer.

	![The Certificate download link](common/certificatebase64download.png)

1. On the **Set up GitHub Enterprise Managed User** section, copy the URLs below and save it for configuring GitHub below.

	![Copy configuration URLs](common/copy-configuration-urls.png)

### Assign the Azure AD test user

In this section, you'll assign your account to GitHub Enterprise Managed User in order to complete SSO set up.

1. In the Azure portal, select **Enterprise Applications**, and then select **All applications**.
1. In the applications list, select **GitHub Enterprise Managed User**.
1. In the app's overview page, find the **Manage** section and select **Users and groups**.
1. Select **Add user**, then select **Users and groups** in the **Add Assignment** dialog.
1. In the **Users and groups** dialog, select your account from the Users list, then click the **Select** button at the bottom of the screen.

> 1. In the **Select a role** dialog, select the **Enterprise Owner** role, then click the **Select** button at the bottom of the screen. This will assign your account as an Enterprise Owner for your GitHub instance.

Reviewer: ```suggestion
1. In the **Select a role** dialog, select the **Enterprise Owner** role, then click the **Select** button at the bottom of the screen. This will assign your account as an Enterprise Owner for your GitHub instance when you provision your account in the next tutorial. 
```
